core/src/main/java/io/grpc/LoadBalancer2.java

```diff
@@ -317,6 +322,77 @@ public String toString() {
   }
 
   /**
+   * Provides arguments for a {@link SubchannelPicker#pickSubchannel(PickSubchannelArgs)}.
+   */
+  public abstract static class PickSubchannelArgs {
```

Comment:
How would you unit test your custom impl of `Picker` without access to builder? Exposing only abstract class would require you to implement it which effectively will be just copy of what's used internally because this class is just a simple container.

Anyway, unclear at this point if you think we should revert it back to `abstract` or make it `public final` (as it is now) ?